ios/RNSScreen.mm

```diff
@@ -101,6 +102,25 @@ - (UIViewController *)reactViewController
 {
   return _reactSubviews;
 }
+
+// taken from how RCTScrollViewComponentView handles sending event for Animated in bridgeless mode
+- (void)dispatchEventForAnimatedObserver:(id<RCTEvent>)event
+{
+  auto eventDispatcher = [[RCTBridge currentBridge] eventDispatcher];
+  if (eventDispatcher) {
+    [eventDispatcher sendEvent:event];
+  } else {
+    id appDelegate = [[UIApplication sharedApplication] delegate];
+    RCTModuleRegistry *moduleRegistry =
+        (RCTModuleRegistry *)[[appDelegate valueForKey:@"_reactHost"] valueForKey:@"_moduleRegistry"];
+    if (moduleRegistry) {
+      id<RCTEventDispatcherProtocol> legacyEventDispatcher = [moduleRegistry moduleForName:"EventDispatcher"
+                                                                     lazilyLoadIfNecessary:YES];
+
+      [legacyEventDispatcher notifyObserversOfEvent:event];
+    }
+  }
+}
```

Comment:
@WoLewicki, you'll have to take the <Image/> component approach for this one as well.

In the ContextContainer, we also provide the RCTEventDispatcher. 

https://github.com/facebook/react-native/blob/f7f9250f6e90a41e798a8f62738925a2859a751a/packages/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm#L270-L277